.editorconfig

```diff
@@ -85,3 +85,52 @@ csharp_new_line_before_catch = true
 csharp_new_line_before_finally = true
 csharp_new_line_before_members_in_object_initializers = true
 csharp_new_line_before_members_in_anonymous_types = true
+
+; Naming styles
+dotnet_naming_style.pascal_case_style.capitalization = pascal_case
+dotnet_naming_style.camel_case_style.capitalization = camel_case
+
+; Naming rule: async methods end in Async
+dotnet_naming_style.async_method_style.capitalization  = pascal_case
+dotnet_naming_style.async_method_style.required_suffix = Async
+dotnet_naming_symbols.async_method_symbols.applicable_kinds = method
+dotnet_naming_symbols.async_method_symbols.required_modifiers = async
+dotnet_naming_rule.async_methods_rule.severity = suggestion
```

Comment:
async unit-test methods don't follow this rule. 
Consider this method for example:
```cs
public async Task CreatePathContextAsync_WithUnrestoredPackagesConfig_Throws()
```